install/linux/usr/share/odemis/sim/fastem-sim.odm.yaml

```diff
@@ -209,11 +209,12 @@ FASTEM-sim: {
 "Beam Shift Controller": {
     class: tfsbc.BeamShiftController,
     role: ebeam-shift,
+    dependencies: {"scanner": "MultiBeam Scanner XT"},
```

Comment:
This actually doesn't work, because the simsem (used in this yaml file but not in the other), does not have the beamshift transformation matrix. Should I just delete this again? Or do you have another suggestion on how to add this?

Reply:
Is the matrix needed for any of the tests where we use this backend?

Reply:
good question, it is not. I will comment this dependency here, and add a comment that this is not supported for the simsem

Reply:
Sounds good!